.github/workflows/test-fakeredis.yml

```diff
@@ -39,18 +36,18 @@ jobs:
       - uses: actions/checkout@v4
         with:
           submodules: true
-      - uses: actions/setup-python@v5
```

Comment:
For some reason this caused a broken installation of python. I fell back to the one provided by ubuntu (3.10 for 22.04)

Reply:
Yes I think the `cache-dependency-path` was the problem but I did not try removing it and keeping the `actions/setup`. We want the the python version that comes with the system and the best way to do that is `apt install` and not relying on some action to fetch the dependency